classes/class-uagb-helper.php

```diff
@@ -892,14 +892,52 @@ public static function get_typography_css( $attr, $slug, $selector, $combined_se
 			);
 		}
 
+		/**
+		 * Sets the selector to Global Block Styles Selector if applicable.
+		 *
+		 * @param string $selector Selector.
+		 * @param array  $gbs_attributes GBS attributes array.
+		 * @since x.x.x
+		 * @return string $selector Updated selector.
+		 */
+		public static function add_gbs_selector_if_applicable( $selector, $gbs_attributes ) {
+			if ( ! empty( $gbs_attributes['globalBlockStyleName'] ) && ! empty( $gbs_attributes['globalBlockStyleId'] ) ) {
+				return self::get_gbs_selector( $gbs_attributes['globalBlockStyleId'] );
+			}
+			return $selector;
```

Comment:
Would this be better?
```suggestion
			if ( empty( $gbs_attributes['globalBlockStyleName'] ) || empty( $gbs_attributes['globalBlockStyleId'] ) ) {
				return $selector;
			}
			return self::get_gbs_selector( $gbs_attributes['globalBlockStyleId'] );
```

Reply:
Changes done